rune/libenclave/internal/runtime/pal/loader.go

```diff
@@ -23,7 +24,8 @@ var fptr_pal_get_local_report unsafe.Pointer
 func Loadbinary(path string) {
 	dl := C.dlopen(C.CString(path), C.RTLD_NOW)
 	if dl == nil {
-		C.perror(C.CString("failed to load library " + path))
+		logrus.Fatal("failed to load %s, dlerror: %s", path, C.GoString(C.dlerror()))
+		return
```

Comment:
logrus.Fatal will call os.Exit(1), so return is not neccessary. Please remove return.

Reply:
removed already. Thanks for pointing it out.